src/main/java/com/patchworkmc/manifest/converter/accesstransformer/AccessTransformerConverter.java

```diff
@@ -0,0 +1,132 @@
+package com.patchworkmc.manifest.converter.accesstransformer;
+
+import java.util.Map;
+
+import net.patchworkmc.manifest.accesstransformer.v2.ForgeAccessTransformer;
+import net.patchworkmc.manifest.accesstransformer.v2.Transformed;
+import net.patchworkmc.manifest.accesstransformer.v2.TransformedClass;
+import net.patchworkmc.manifest.accesstransformer.v2.TransformedField;
+import net.patchworkmc.manifest.accesstransformer.v2.TransformedMethod;
+import net.patchworkmc.manifest.accesstransformer.v2.TransformedWildcardMember;
+import net.patchworkmc.manifest.accesstransformer.v2.flags.AccessLevel;
+import net.patchworkmc.manifest.accesstransformer.v2.flags.Finalization;
+import com.patchworkmc.mapping.IntermediaryHolder;
+
+/**
+ * Takes a {@link ForgeAccessTransformer} and spits out an AccessWidener/v1 file
+ */
+public class AccessTransformerConverter {
+	private AccessTransformerConverter() {
+		// NO-OP
+	}
+
+	public static byte[] convertToWidener(ForgeAccessTransformer accessTransformer, IntermediaryHolder holder) {
+		StringBuilder sb = new StringBuilder();
+		sb.append("accessWidener\tv1\t").append("intermediary").append('\n');
+		sb.append("#\tAutomatically generated by Patchwork Patcher.\n");
+		sb.append("#\tThe original (but remapped) AT entry is roughly preserved here for debugging.\n");
+		sb.append("#\tThey may be stripped in future releases.\n\n");
+
+		for (TransformedClass targetClass : accessTransformer.getClasses()) {
+			if (targetClass.getAccessLevel() != AccessLevel.KEEP || targetClass.getFinalization() != Finalization.KEEP) {
+				writeClass(sb, targetClass);
+			}
+			writeWildcards(sb, targetClass.getName(), (TransformedWildcardMember) targetClass.getDefaultForFields(),
```

Comment:
I don't like the casts here, is there a way to avoid this? Maybe patchwork-manifest should have a better interface here